seedemu/core/Node.py

```diff
@@ -408,6 +408,10 @@ def __joinNetwork(self, net: Network, address: str = "auto"):
         """
         
         if address == "auto": _addr = net.assign(self.__role, self.__asn)
+        elif address == "dhcp": 
+            _addr = None
+            self.addSoftware('isc-dhcp-client')
+            self.appendStartCommand('ip addr flush {iface}; dhclient;'.format(iface=net.getName()))
```

Comment:
doesn't running `dhclient` w/o parameters makes it run on all interfaces? if this is not intentional, consider setting interface for `dhclient`: 
```suggestion
            self.appendStartCommand('ip addr flush {iface}; dhclient {iface};'.format(iface=net.getName()))
```

Reply:
I think specifying an interface for `dhclient` is needed. Thanks for your suggestion.